example/lock.sbt

```diff
@@ -0,0 +1,113 @@
+// DON'T EDIT THIS FILE.
+// This file is auto generated by sbt-lock 0.6.1.
+// https://github.com/tkawachi/sbt-lock/
+dependencyOverrides ++= {
```

Comment:
i don't like this at all - what problem does it solve?

Reply:
I also think right now we are not in a state where we would need this. We can always add it but I'd like to remove it for now too.

Reply:
let's not worry about `lock` right now. removed.